src/index.html

```diff
@@ -4,9 +4,374 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="/" class="nav__item is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/"
+            class="nav__item"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Фото
+            </a>
+          </li>
```

Comment:
```suggestion
          </li>
          
          <li class="nav__li">
            <a href="/" class="nav__item">
              Фото
            </a>
          </li>
```
My suggestion to do extra space between your lis for more comfortable reading your code